### PR TITLE
WIP: F 2400 gtfs exclude fixed links and vstp

### PR DESCRIPTION
**Pull request by iblocks-Cheng**
Add a new web command parameter to give us option exclude fixed links: https://issues.iblocks.co.uk/browse/SMARTTIS-2400

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (iblocks-Cheng) [inline .vscode/launch.json]: I think I have to remove this file before final merging, am I right?

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (iblocks-Cheng) [inline .vscode/tasks.json]: and this file? I think I have to remove this file?

File: package-lock.json [hunk 3]
```diff
@@ -68,6 +68,15 @@
         "@types/node": "*"
```
Comment (iblocks-Cheng) [inline package-lock.json]: do I have to roll back this `package-lock.json` file before merging?

**Comment by linusnorton** [comment]
@iblocks-Cheng Hi Cheng, there are a number of changes in here that probably need discussion - are you able to break it down into smaller PRs?

**Comment by iblocks-Cheng** [comment]
Hi Linus, sorry that I started an incorrect pull request (new to GitHub.. ), this branch was meant to merge into another branch and I have closed the pull request.

**Pull request closed without merging**